jekyll/_includes/snippets/logic-statement-examples/when-in-workflows/pr-synced-or-labeled.yml

```diff
@@ -0,0 +1,4 @@
+workflows:
+# The "run_on_pr" workflow only runs when open pull requests are labeled or updated with new commits.
+  run_on_pr:
+    when: (pipeline.event.name == "pull_request" and pipeline.event.action == "labeled") or (pipeline.event.name == "pull_request" and pipeline.event.action == "synchronize")
```

Comment:
Please add newline at end of files. (There's probably a setting in your editor/IDE that can make this permanent.)